src/renderer/js/component/shapeShift/internal/active-shift.jsx

```diff
@@ -0,0 +1,147 @@
+import type { Dispatch } from "redux/actions/shape_shift";
+
+type Props = {
+  shiftState: ?string,
+  shiftCoinType: ?string,
+  shiftDepositAddress: ?string,
+  shiftReturnAddress: ?string,
+  shiftOrderId: ?string,
+  originCoinDepositMax: ?number,
+  clearShapeShift: Dispatch,
+  doShowSnackBar: Dispatch,
+  getActiveShift: Dispatch,
+};
+
+class ActiveShapeShift extends React.PureComponent<Props> {
+  constructor() {
+    super();
+    // $FlowFixMe
+    this.continousFetch = undefined;
+  }
+
+  componentDidMount() {
+    const { getActiveShift, shiftDepositAddress } = this.props;
+
+    getActiveShift(shiftDepositAddress);
+    // $FlowFixMe
+    this.continousFetch = setInterval(() => {
+      getActiveShift(shiftDepositAddress);
+    }, 10000);
+  }
+
+  componentWillUpdate(nextProps: Props) {
+    const { shiftState } = nextProps;
+    if (shiftState === statuses.COMPLETE) {
+      this.clearContinuousFetch();
+    }
+  }
+
+  componentWillUnmount() {
+    this.clearContinuousFetch();
+  }
+
+  clearContinuousFetch() {
+    /// $FlowFixMe
+    clearInterval(this.continousFetch);
+    // $FlowFixMe
+    this.continousFetch = null;
+  }
+
+  render() {
+    const {
+      shiftCoinType,
+      shiftDepositAddress,
+      shiftReturnAddress,
+      shiftOrderId,
+      shiftState,
+      originCoinDepositMax,
+      clearShapeShift,
+      doShowSnackBar,
+    } = this.props;
+
+    return (
+      <div>
+        {shiftState === statuses.NO_DEPOSITS && (
+          <div>
+            <p>
+              Send up to{" "}
+              <span className="credit-amount--bold">
+                {originCoinDepositMax}{" "}
+                <span className="credit-amount--colored">{shiftCoinType}</span>
+              </span>{" "}
+              to the address below.
+            </p>
+
+            <div className="shapeshift__deposit-address-wrapper">
+              <Address address={shiftDepositAddress} showCopyButton />
+              <div className="shapeshift__qrcode">
+                <QRCode value={shiftDepositAddress} />
+              </div>
+            </div>
+          </div>
+        )}
+
+        {shiftState === statuses.RECEIVED && (
+          <div className="card__content--extra-vertical-space">
+            <p>
+              {__(
+                "ShapeShift has received your payment! Sending the funds to your LBRY wallet."
+              )}
+            </p>
+            <span className="help">
+              {__("This can take a while, especially with BTC.")}
+            </span>
+          </div>
+        )}
+
+        {shiftState === statuses.COMPLETE && (
+          <div className="card__content--extra-vertical-space">
+            <p>
+              {__(
+                "Transaction complete! You should see the new LBC in your wallet."
+              )}
+            </p>
+          </div>
+        )}
+        <div className="shapeshift__actions">
+          <Link
+            button="primary"
```

Comment:
Should be "alt" for Cancel.

Reply:
What happened to the idea of a "Sent" button that displays the shift code for tracking purposes?

Reply:
I'm not sure how much I like that button. What if they click it before actually sending? We would need some new bit of UI to say "We will keep pinging shapeshit to see if you actually sent it" (obviously worded better).

We say "this will update automatically" and have a link to the shapeshift status page, I don't see any reason to create another shift status "user says they sent it but shapeshift hasn't received it"